src/styles/reset.scss

```diff
@@ -0,0 +1,8 @@
+body,
+p {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
It's not reset, it's normalize, create file for this and move there styles for class main too